main.py

```
@@ -20,23 +25,81 @@ def health():
 
 
 @app.post("/yolo/predict")
-async def predict(file: UploadFile):
-    # 이미지 전처리
-    image = Image.open(io.BytesIO(await file.read())).convert("RGB").resize((640, 640))
+async def predict(
+    file: Optional[UploadFile] = File(None), url: Optional[str] = Form(None)
```

Ruff (0.12.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)